nixos/tests/hydra/default.nix

```diff
@@ -0,0 +1,75 @@
+import ../make-test.nix ({ pkgs, ...} :
+
+let
+   trivialJob = pkgs.writeTextDir "trivial.nix" ''
+     with import <nix/config.nix>;
+
+     { trivial = builtins.derivation {
+         name = "trivial";
+         system = "x86_64-linux";
+         PATH = coreutils;
+         builder = shell;
+         args = ["-c" "touch $out"];
+       };
+     }
+   '';
+
+    createTrivialProject = pkgs.stdenv.mkDerivation {
+      name = "create-trivial-project";
+      unpackPhase = ":";
+      buildInputs = [ pkgs.makeWrapper ];
+      installPhase = "install -m755 -D ${./create-trivial-project.sh} $out/bin/create-trivial-project.sh";
+      postFixup = ''
+        wrapProgram "$out/bin/create-trivial-project.sh" --prefix PATH ":" ${pkgs.stdenv.lib.makeBinPath [ pkgs.curl ]} --set EXPR_PATH ${trivialJob}
+      '';
+    };
+
+in {
+  name = "hydra-init-localdb";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ pstn lewo ];
+  };
+
+  machine =
+    { config, pkgs, ... }:
+
+    {
+      virtualisation.memorySize = 1024;
+      environment.systemPackages = [ createTrivialProject pkgs.jq ];
```

Comment:
Adding `time.timeZone = "UTC";` here will eliminate ugly time zone warnings from the test log.

Reply:
I'm not able to find these warnings. Could you paste an example? Are they in the nix-build output or in the generated html test log?

Reply:
in both, see here: https://gist.github.com/xeji/32c3f495a91f4538b4ba000fc3220582#file-hydra-test-log-L954

Reply:
Ok, fixed thanks (this is only visible with sandboxed builds).